src/System.Composition.Hosting/src/System/Composition/Hosting/Util/SmallSparseInitonlyArray.cs

```diff
@@ -38,7 +35,7 @@ public void Add(int index, object value)
 
             if(e.Index == index)
             {
-                throw new ArgumentException(String.Format(CultureInfo.CurrentCulture, SR.Key_Already_Exist, index), nameof(index));
+                throw new ArgumentException(SR.Format(SR.Key_Already_Exist, nameof(index)), nameof(index));
```

Comment:
are you sure that the nameof here and below is correct? I believe the error message prints out the index which already exists. Currently you always log "index".

Reply:
Oops. i put `nameof` there by mistake.